pillarbox-core-business/src/androidTest/java/ch/srgssr/pillarbox/core/business/LocalMediaCompositionDataSource.kt

```diff
@@ -13,10 +13,11 @@ class LocalMediaCompositionDataSource(context: Context) : MediaCompositionDataSo
     private val localData = HashMap<String, MediaComposition>()
 
     init {
-        val jonsSerializer = Json { ignoreUnknownKeys = true }
-        for (urn in urns) {
```

Comment:
`urns` is not used anymore, and can be removed.
The other fields in the `companion object` are only used in `CommandersActTrackerTest`, and can probably be moved there?

Reply:
Yes right we can removes `urns` but I will still keep other hardcoded urns for tests here. Has they are linked to the json file that we load in this class.